networkit/dynbase.pyx

```diff
@@ -0,0 +1,80 @@
+# distutils: language=c++
+
+from libcpp.cast cimport dynamic_cast
+
+from .dynamics cimport _GraphEvent, GraphEvent
+from .base cimport _CythonParentClass
+from typing import List
+
+ctypedef _DynAlgorithm* _DynAlgorithmPtr
+
+cdef class _CythonSubclassDynAlgorithm(_CythonParentClass):
+	""" Abstract base class for dynamic algorithms
+    Cython does not allow direct inheritance from two extension types (cdef classes).
```

Comment:
Indentation for the the docstring looks of. Also move the closing `"""` to another line.